cms/djangoapps/contentstore/api/views/course_quality.py

```diff
@@ -149,7 +148,7 @@ def _sections_quality(self, course):
             total_visible=len(visible_sections),
             number_with_highlights=len(sections_with_highlights),
             highlights_active_for_course=course.highlights_enabled_for_messaging,
-            highlights_enabled=highlights_setting.is_enabled(),
+            highlights_enabled=True,  # used to be controlled by a waffle switch, now just always enabled
```

Comment:
Here in this API and in the xblock info code, I chose to leave these fields defined, just set to a constant. I didn't want to break any api assumptions.